src/core/dfn-panel.js

```diff
@@ -0,0 +1,161 @@
+// Constructs "dfn panels" which show all the local references to a dfn and a
+// self link to the selected dfn. Based on Bikeshed's dfn panels at
+// https://github.com/tabatkins/bikeshed/blob/ef44162c2e/bikeshed/dfnpanels.py
+import { fetchAsset } from "./text-loader.js";
+import { hyperHTML } from "./import-maps.js";
+import { norm } from "./utils.js";
+
+const cssPromise = loadStyle();
+
+export const name = "core/dfn-panel";
+
+export async function run() {
+  const css = await cssPromise;
+  document.head.insertBefore(
+    hyperHTML`<style>${css}</style>`,
+    document.querySelector("link")
+  );
+
+  document.body.addEventListener("click", event => {
+    /** @type {HTMLElement} */
+    const el = event.target;
+    const panel = document.getElementById("dfn-panel");
+
+    const action = parseAction(el);
+    switch (action) {
+      case "SHOW": {
+        if (panel) panel.remove();
+        const dfn = el.closest("dfn");
+        displayPanel(dfn, createPanel(dfn));
+        break;
+      }
+      case "ACTIVATE": {
+        panel.classList.add("activated");
+        panel.style.left = null;
```

Comment:
no way to do this with CSS, I guess?

Reply:
nothing beats inline styles added via JS

Reply:
oh oh.. we can use CSS variables :smile: